assets/helpers/url.js

```diff
@@ -92,6 +92,14 @@ function getBaseDomain(): Domain {
 
 }
 
+function getAbsoluteURL(path: string): string {
+  const loc = window.location;
+  const origin = window.location.origin ||
+    `${loc.protocol}//${loc.hostname}${loc.port ? `:${loc.port}` : ''}`;
```

Comment:
This is identical to what's in `getBaseDomain` above, could you pull out into a function?